### ISSUE TITLE
Does not install in linux

### ISSUE THREAD

**Issue by ghost** [Acolarh/colorgrab#10]

**Comment by BoAnd** [comment]
"Does not install in linux" does not help the developer fixing the problem. So maybe try this template:
### Expected behavior
### Actual behavior
### Steps to reproduce the bahavior
Or maybe I can fix the problem then. 

Best regards BoAnd

**Comment by P4ulJC** [comment]
Hi,

The install failed for me on CentOS 8 (x86_64 )  with the below error. I'm installing from the /tmp directory.

/tmp//src/tools/PaletteTool.cpp: In member function ‘void PaletteTool::AddColor(const wxColour&, const wxString&)’:
/tmp/colorgrab/src/tools/PaletteTool.cpp:176:35: error: call of overloaded ‘wxVariant(const wxColour&)’ is ambiguous
     data.push_back(wxVariant(color));

In file included from /usr/include/wx-3.0/wx/any.h:605,
                 from /usr/include/wx-3.0/wx/event.h:29,
                 from /usr/include/wx-3.0/wx/window.h:18,
                 from /usr/include/wx-3.0/wx/nonownedwnd.h:14,
                 from /usr/include/wx-3.0/wx/toplevel.h:20,
                 from /usr/include/wx-3.0/wx/frame.h:18,
                 from /tmp/colorgrab/src/ToolWindow.h:9,
                 from /tmp/colorgrab/src/tools/PaletteTool.h:9,
                 from /tmp/colorgrab/src/tools/PaletteTool.cpp:6:
/usr/include/wx-3.0/wx/variant.h:187:5: note: candidate: ‘wxVariant::wxVariant(bool, const wxString&)’
     wxVariant(bool val, const wxString& name = wxEmptyString);

**Comment by nielssp** [comment]
@P4ulJC Could you try building the [develop](https://github.com/nielssp/colorgrab/tree/develop) branch instead? I think I've seen the error before and fixed it in [this commit](https://github.com/nielssp/colorgrab/commit/5ecc8e082197e5205fbe365e4897309967d7c26c).

**Comment by P4ulJC** [comment]
I tried building the develop branch and it compiled without errors! But unfortunately I can't use it because no colour pickers work on CentOS 8. I've tried 5 different ones! The picker does not work outside of the app window - so it can't pick anything apart from the app itself (and even then colours are not picked - same with all apps). This is common to all programs - including the Gimp's colour picker. It's related to Wayland I think - I know nothing about Wayland apart from the fact you can't use colour pickers with it!

**Comment by nielssp** [comment]
Yeah, unfortunately I don't think Wayland allows applications to take screenshots/read pixels from the screen. Last time I looked into it I couldn't find a way around that.

**Comment by P4ulJC** [comment]
Yes you are right about Wayland and I wish I'd known that before I started installing 5 different colour pickers! I think Grim is the thing to use with Wayland to grab the screen. I'm part way through installing dependencies for it - but have given up for the moment after  a dependency of a dependency needed updating and I couldn't get the new version to install without a few other dependencies.... I think it's called Grim because it's such a grim task trying to install it...

**Comment by roke-julian-lockhart** [comment]
If the problem is caused by Wayland and/or CentOS, why not report this problem to the developers of those projects and consequently close this issue?

**Comment by P4ulJC** [comment]
I forgot all about this because I ditched Wayland the following day due to how many issues I was having.  Apologies for leaving it so long. I know nothing about Github apart from making the occasional comment like this - not sure if I close the issue  or what...
